tests/duplicates/test_duplicates.py

```diff
@@ -1,58 +1,64 @@
-# Copyright (C) 2021 Intel Corporation
+# Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 
-import unittest
 import logging
-from codebasin import config, finder, walkers
+import unittest
+from pathlib import Path
+
+from codebasin import finder
 from codebasin.walkers.platform_mapper import PlatformMapper
 
 
-class TestExampleFile(unittest.TestCase):
+class TestDuplicates(unittest.TestCase):
     """
     Test ability to detect and report identical files in a codebase.
-    Such duplicates may appear in out-of-tree builds and can skew results.
+    Such duplicates SHOULD count towards divergence.
```

Comment:
I feel like this should be exclaimed a little bit louder, but I don't know where and how is best...

Reply:
I've moved the comment in e77e6f548f0dd9e57172f53e855699d1e3315ed6.

This might not look like it does much, but I recently discovered that `unittest --verbose` prints the docstring for the test.  So, after this change the output contains:
```
test_duplicates (tests.duplicates.test_duplicates.TestDuplicates.test_duplicates)
Check that duplicate files count towards divergence. ... ok
```

...which is a little louder than before.  I've opened #84 to track a dedicated feature to report the presence of duplicate files.